tests/tx/test_prometheus.py

```diff
@@ -14,6 +17,7 @@ def setUp(self):
         self.network = 'testnet'
         self.manager = self.create_peer(self.network, unlock_wallet=True)
 
+    @pytest.mark.skipif(sys.platform == 'win32', reason='set_new_metrics fails on Windows')
```

Comment:
Do you know the error message? I don't see anything obvious that might raise an exception.

Reply:
I actually haven't debugged it at all. I don't know if it's worth the time, I think we can leave this to a separate task.

Reply:
Same as above.